appeals/web/src/server/appeals/appeal-details/costs/costs.mapper.js

```diff
@@ -40,53 +39,74 @@ export function addDocumentTypePage(appealDetails, documentTypes) {
 }
 
 /**
+ *
  * @param {import('../appeal-details.types.js').WebAppeal} appealDetails
- * @param {string} costsApplicant
- * @param {string} folderId
- * @param {string} folderPath
- * @param {string} documentType
- * @param {import('@pins/express').ValidationErrors|undefined} errors
- * @returns {import('#appeals/appeal-documents/appeal-documents.types.js').DocumentUploadPageParameters}
+ * @param {import('@pins/appeals.api').Schema.Folder} decisionDocumentFolder
+ * @returns {PageContent}
  */
-export function documentUploadPage(
-	appealDetails,
-	costsApplicant,
-	folderId,
-	folderPath,
-	documentType,
-	errors
-) {
-	const pathComponents = folderPath.split('/');
-	const documentStage = pathComponents[0];
-
-	let costsApplicantLabel = '';
-
-	switch (costsApplicant) {
-		case 'appellant':
-			costsApplicantLabel = 'Appellant ';
-			break;
-		case 'lpa':
-			costsApplicantLabel = 'LPA ';
-			break;
-	}
+export function decisionCheckAndConfirmPage(appealDetails, decisionDocumentFolder) {
+	const shortAppealReference = appealShortReference(appealDetails.appealReference);
 
-	const pageTitleText = `Upload ${costsApplicantLabel}costs document`;
+	/** @type {PageComponent} */
+	const summaryListComponent = {
+		type: 'summary-list',
+		parameters: {
+			rows: [
+				{
+					key: {
+						text: 'Costs decision'
+					},
+					value: {
+						html: `<ul class="govuk-list"><li>${(decisionDocumentFolder.documents || [])
```

Comment:
Do we not have a formatter to create lists? Would it be worth creating one?

Reply:
I thought this might be more common, but apparently not. Could be dealt with in some refactoring in the future if it becomes more common pattern?

Reply:
There are a few in `appeals/web/src/server/lib/display-page-formatter.js` but they are all use-case specific - as you say, it's something we can look at refactoring as an improvement